src/main/java/Main.java

```diff
@@ -0,0 +1,43 @@
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
+    /**
+     * Creates Duke.
+     */
+    private final Duke duke = new Duke();
+    /**
+     * Gets the image for Duke.
```

Comment:
you can just use /* */ for one line JavaDoc

Reply:
Okay thank you for pointing it out!